Procfile

```diff
@@ -0,0 +1,2 @@
+vue-builder: vue-cli-service build --mode development --watch
```

Comment:
What do you think about using a shorter name like `build` or `vue-build`?

Reply:
Fine with any option :slightly_smiling_face:   Maybe even just `vue`?